airflow/operators/docker_operator.py

```diff
@@ -247,7 +247,7 @@ def execute(self, context):
                     if self.xcom_all else line.encode('utf-8')
 
     def get_command(self):
-        if self.command is not None and self.command.strip().find('[') == 0:
+        if isinstance(self.command, str) and self.command.strip().find('[') == 0:
```

Comment:
Why do we have this `and self.command.strip().find('[') == 0`?

Just trying to understand the logic.

Reply:
When Jinja returns the "[" sign, an array is created. Please look at related PR: https://github.com/apache/airflow/pull/4900

Reply:
Jinja would convert a list to list. `self.command.strip().find('[') == 0` is checking a **string**, right? and checking if the first character is `[`.

Sorry, maybe I am missing something. @mik-laj Feel free to merge this PR :)

Reply:
When a string is encountered, but the string looks like it contains arrays (starts with the character "[") it is processed by literal_parse, which creates the real table from the string.
```
>>> import ast
>>> ast.literal_eval('["A", "B"]')
['A', 'B']
>>>
```

Currently, Jinja always accepts a string at the input and returns a string. However, the mentioned PR discussed an alternative solution to the one used in this PR.

I can not do it at the moment. I just wanted to answer your question.

Reply:
@kaxil @mik-laj So would you like to see any changes in this PR or does it have your approval?

Reply:
Thanks, @mik-laj for the explanation. I understand it now.